### DISCUSSION TITLE
Add "Remove" button to suppression list

### DISCUSSION THREAD

**Discussion by sorbanbela** [postalserver/postal#2900] [Feature suggestions]
We have been removing addresses from the suppression list using a database management software but I think it would be nice to have a Remove button which should give the address a new chance to work (for ex. if the probleb has been resolved).

**Comment by willpower232** [comment]
Suppressed email addresses can be removed if you manually release a held message and the message is successfully delivered.

**Reply by WelshDev** [reply]
The problem with this is that I don't want to redeliver the message that's been held. We're using Postal to send out time-sensitive emails which contain links that expire after just a couple of hours. By the time we realise the delivery has failed and it's on the suppression list that email is no longer relevant.

I'd really love to have the ability to quickly and easily remove an email from the suppression list.
